schemas/v1.0/azure.yaml.json

```diff
@@ -135,9 +140,125 @@
                                 "description": ""
                             }
                         }
+                    },
+                    "scripts": {
+                        "type": "object",
+                        "title": "Definition of command hooks for developer integration",
+                        "descriptions": "Command hooks should match `azd` command names prefixed with `pre` or `post` depending on when the script should execute relative to the command",
+                        "minProperties": 1,
+                        "additionalProperties": {
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                                "type": {
+                                    "type": "string",
+                                    "title": "Type of script",
+                                    "description": "Optional. The type of script to use for the command hook. (Default: bash)",
```

Comment:
Should PowerShell be default on Windows, bash on MacOS/Linux?

Reply:
I feel that since most CI runners will run on linux workloads we should default to bash.